src/test/resources/namespaces.properties

```diff
@@ -0,0 +1,4 @@
+dcterms = http://purl.org/dc/terms/
```

Comment:
There's a really strange behavior in fcrepo4 that I think we should talk about in the comments for this file.  Could you add something like the following to the top of this file:

# The first time a property is added to the repository belonging to a particular namespace, that namespace 
# prefix is registered and will be used from that point on (for all properties using that namespace).  Setting values
# here *before the first run* will allow registration of preferred namespace prefixes, but will have no effect on
# subsequent runs.